src/data/markdown/translated-guides/en/03 Using k6 browser/100 Examples.md

```diff
@@ -0,0 +1,10 @@
+---
+title: 'Examples'
+excerpt: 'A list of different examples and recommended practices when working with the k6 browser module'
+---
+
+This section presents some examples and recommended practices when working with the `k6 browser` module to leverage browser automation as part of your k6 tests.
+
+- [Page object model pattern](/using-k6-browser/examples/page-object-model-pattern)
```

Comment:
```suggestion
- [Page object model pattern](/using-k6-browser/examples/page-object-model)
```

It looks like the link should be `page-object-model` here.

Reply:
Yes, thanks @heitortsergent! 💜